### PR TITLE
Feature/implement helper sage replacement functions

### PR DESCRIPTION
**Pull request by Dioprz**
### Description

**Main changes**

Python implementation of the `is_prime_power` and `is_power_of_two`functions, which will replace its homonymous Sage functions.

- `is_power_of_two`

  - Uses bitwise operations for faster computations.

  - Performance changes over Sage: 7x faster.

- `is_prime_power`

  -  Uses a precomputed list for small inputs, and uses numbers with the shape $6k\pm 1$ (prime candidates) to check divisibility on larger inputs.

  - Performance changes over Sage:

    - On inputs in the range of the precomputed primes: Same performance.

    - On larger inputs: About 1.5 times slower than Sage. This is a natural implication when comparing a Python function versus a C function from a highly optimized library (such as the one used by Sage, via the **PARI** library).

**Minor changes**

- Stylistic changes via Black.
- Remove one unused import.

### Review process

- **Feedback needed:** Do you agree to put this kind of functions in the `helper.py` file, or should I create a new file for that?
- Functions have been already intensively tested on my side.

### Pre-approval checklist
- [x] The code builds clean without any errors or warnings
- [ ] I've added/updated tests
- [x] I've added/updated documentation if needed

### CODE REVIEW

File: cryptographic_estimators/helper.py [hunk 1]
```diff
@@ -82,6 +82,79 @@ def round_or_truncate(x: float, truncate: bool, precision: int):
...
+    509, 521, 523, 541, 547, 557, 563, 569, 571, 577, 587, 593, 599, 601, 607,
+    613, 617, 619, 631, 641, 643, 647, 653, 659, 661, 673, 677, 683, 691, 701,
+    709, 719, 727, 733, 739, 743, 751, 757, 761, 769, 773, 787, 797, 809, 811,
+    821, 823, 827, 829, 839, 853, 857, 859, 863, 877, 881, 883, 887, 907, 911,
+    919, 929, 937, 941, 947, 953, 967, 971, 977, 983, 991, 997
+]
+# fmt: on
+
+
+def is_prime_power(n):
```
Comment (Javierverbel) [inline cryptographic_estimators/helper.py:106]: Maybe it is good idea to add several tests to these functions

**Review by Javierverbel — CHANGES_REQUESTED** [review]

File: cryptographic_estimators/helper.py (hunk 1 above)
Comment (Dioprz) [reply cryptographic_estimators/helper.py:110]: Done!

**Review by Javierverbel — APPROVED** [review]

File: cryptographic_estimators/helper.py [hunk 2]
```diff
@@ -60,20 +63,20 @@ def concat_pretty_tables(t1: str, t2: str):
...
 
     """
-    return float(int(x * 10 ** precision) / 10 ** precision)
+    return float(int(x * 10**precision) / 10**precision)
 
 
 def round_or_truncate(x: float, truncate: bool, precision: int):
     """
-    eiter rounds or truncates `x` if `truncate` is `true`
+    Eiter rounds or truncates `x` if `truncate` is `true`
```
Comment (FloydZ) [inline cryptographic_estimators/helper.py:79]: Either

**Review by FloydZ — APPROVED** [review]
Nice work!

File: cryptographic_estimators/helper.py (hunk 2 above)
Comment (Dioprz) [reply cryptographic_estimators/helper.py:79]: Huh, I missed that error.  Thanks! I will fix this typo in my next PR, so we can merge this right now.

**Pull request merged**